flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/service/AbstractFlinkService.java

```diff
@@ -160,19 +163,31 @@ public KubernetesClient getKubernetesClient() {
         return kubernetesClient;
     }
 
+    public CuratorFrameworkWithUnhandledErrorListener getCurator(Configuration conf) {
+        return ZooKeeperUtils.startCuratorFramework(conf, exception -> {});
```

Comment:
Note that by leveraging the [ZooKeeperUtils::startCuratorFramework](https://github.com/apache/flink/blob/c57b84921447bb0ade5e1ff77a05ebd8bbbe71b7/flink-runtime/src/main/java/org/apache/flink/runtime/util/ZooKeeperUtils.java#L200) function and by taking the application `conf`, we create the client in exactly the same way as the Flink application itself. This means that we should be able to support all ZK configuration settings that are supported by Flink applications themselves (ex: ACLs).

Reply:
👍